cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -135,6 +140,10 @@ func run(*cobra.Command, []string) {
 		log.Warn().Msgf("--no-migration flag is deprecated")
 	}
 
+	if flagValidateMigration {
+		log.Warn().Msgf("validation flag is enabled and will increase duration of migration")
```

Comment:
```suggestion
		log.Warn().Msgf("atree migration validation flag is enabled and will increase duration of migration")
```
I would be a bit more specific here.